stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -141,6 +147,7 @@ internal class ChatDomainImpl internal constructor(
     private val _banned = MutableStateFlow(false)
     private val _mutedUsers = MutableStateFlow<List<Mute>>(emptyList())
     private val _typingChannels = MediatorLiveData<TypingEvent>()
+    private val useCaseProvider: UseCaseHelper = UseCaseHelper(this)
```

Comment:
Why did you decide to create new instance? Why can't we use just deprecated `useCaseHelper`?

Reply:
Because it will be deprecated with `DeprecationLevel.Error` soon